src/editing/editFeatureComponent.js

```diff
@@ -713,12 +713,25 @@ Controller.prototype.$onInit = function () {
   );
 
   // (3) Get attributes
-  this.gmfXSDAttributes_.getAttributes(this.editableNode_.id).then(this.setAttributes_.bind(this));
+  this.gmfXSDAttributes_.getAttributes(this.editableNode_.id).then((attributes) => {
+    this.setAttributes_(attributes);
+    this.setLayerStyle();
+  });
 
   // (4) Toggle
   this.toggle_(true);
 };
 
+/**Set the style on the editable layer */
+Controller.prototype.setLayerStyle = function () {
+  if (!this.vectorLayer) {
+    throw new Error('Missing vectorLayer');
+  }
+  if (this.attributes.find((attribute) => attribute.name === 'geom').readonly) {
```

Comment:
You are sure that's always geom?

Reply:
In data source we have a method `geometryName`.

Reply:
Not sure that's used in editing!

Reply:
There is a method getGeometryAttribute(this.attributes) already used in editing which works well...